timeseries/src/autogluon/timeseries/models/sktime/models.py

```diff
@@ -59,8 +194,79 @@ class AutoETSModel(AbstractSktimeModel):
         "random_state",
     ]
 
+    def _get_skt_forecaster_args(self, args: dict, min_length: int, inferred_period: int = 1):
+        fail_if_misoconfigured = args.pop("fail_if_misconfigured", False)
+
+        if min_length < 8 and args.get("initialization_method") == "heuristic":
+            error_message = f"{self.name}: Training series too short for initialization_method='heuristic'."
+            if fail_if_misoconfigured:
+                raise ValueError(error_message)
+            else:
+                warnings.warn(error_message + "\nFalling back to initialization_method='estimated'")
+                args["initialization_method"] = "estimated"
+
+        seasonality = args.pop("seasonality", "add")
+        if seasonality not in ["add", "mul", None]:
+            raise ValueError(
+                f"Invalid seasonality {seasonality} for model {self.name} (must be one of 'add', 'mul', None)"
+            )
+
+        seasonal_period = args.pop("seasonal_period", None)
+        # Handle the case when seasonal_period is intentionally set to None
+        if seasonal_period is None:
+            seasonal_period = inferred_period
```

Comment:
what happens if `seasonality is None` but inferred period is not 1?

Reply:
If we set `seasonal_period` but `seasonality=None`, seasonality will be turned off. This is also true for other sktime models (except TBATS, where setting `seasonal_period != 1` is enough to turn on seasonality).